consensus/hotstuff/signature.go

```diff
@@ -96,5 +96,8 @@ type Packer interface {
 
 	// blockID is the block that the aggregated sig is signed for
 	// sig is the aggregated signature data
+	// It returns:
+	//  - (sigData, nil) if successfully unpacked the signature data
+	//  - (nil, signature.ErrInvalidFormat) if failed to unpack the signature data
```

Comment:
this sentinel error is useful for [event handler to handler errors from the validator](https://github.com/onflow/flow-go/blob/master/consensus/hotstuff/validator/validator.go#L65-L67)